apps/product/tasks.py

```
@@ -1,207 +1,982 @@
+
+@app.task
+def update_wildberries_stocks():
+    
+    for wildberries in Wildberries.objects.all():
+        wb_api_key = wildberries.wb_api_key
+        response = requests.get(wildberries_stocks_url, headers={'Authorization': f'Bearer {wb_api_key}'})
+
+        for item in response.json():
+            try:
+                warehouse = item['warehouseName']
+            except:
```

Ruff

Do not use bare `except`

(E722)